packages/retail-ui-extensions/src/extension-api/standard-api.ts

```diff
@@ -1,9 +1,11 @@
 import type {CartApi} from './cart-api';
 import type {LocaleApi} from './locale-api';
 import type {SessionApi} from './session-api';
+import {ToastApi} from './toast-api';
 
 export type StandardApi<T> = {[key: string]: any} & {
   extensionPoint: T;
 } & LocaleApi &
   CartApi &
+  ToastApi &
```

Comment:
@NathanJolly I think we forgot to include the ToastApi in the api object interface. We probably want the ToastApi to be available on both extension points right?

Reply:
Thank you! Correct: Both extension points.